<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adminWindow</class>
 <widget class="QMainWindow" name="adminWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>管理员</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>471</width>
      <height>421</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="adDishesTab">
     <attribute name="title">
      <string>菜品管理</string>
     </attribute>
     <widget class="QTableView" name="adDishesTabView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>441</width>
        <height>371</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="adPeopleTab">
     <attribute name="title">
      <string>人员管理</string>
     </attribute>
     <widget class="QTableView" name="adPeopleTabView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>441</width>
        <height>371</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="adOrderTab">
     <attribute name="title">
      <string>订单管理</string>
     </attribute>
     <widget class="QTableView" name="adOrderTabView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>441</width>
        <height>371</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="adDayTab">
     <attribute name="title">
      <string>当日营业额</string>
     </attribute>
     <widget class="QTableView" name="adDayTabView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>441</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="adDaySumLab">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>340</y>
        <width>441</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>25</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>总计 : </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="adMonthTab">
     <attribute name="title">
      <string>当月营业额</string>
     </attribute>
     <widget class="QTableView" name="adMonthTabView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>441</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="adMonthSumLab">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>340</y>
        <width>441</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>25</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>总计 : </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="adDateTab">
     <attribute name="title">
      <string>全部营业额</string>
     </attribute>
     <widget class="QTableView" name="adDateTabView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>441</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="adDateSumLab">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>340</y>
        <width>441</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>25</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>总计 : </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="adAddButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>470</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>增</string>
    </property>
   </widget>
   <widget class="QPushButton" name="adDelButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>470</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>删</string>
    </property>
   </widget>
   <widget class="QPushButton" name="adChButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>470</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>改</string>
    </property>
   </widget>
   <widget class="QPushButton" name="adSelButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>470</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>查</string>
    </property>
   </widget>
   <widget class="QPushButton" name="adPwdButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>310</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>修改密码</string>
    </property>
   </widget>
   <widget class="QPushButton" name="adSubButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>470</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>提交</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>50</y>
      <width>271</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="adNameLab">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>用户名 :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="adNameEdit">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Normal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="adPwd1Lab">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>原密码 :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="adPwd1Edit">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="adPwd2Lab">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>新密码 :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="adPwd2Edit">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="adPwd3Lab">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>新密码 :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="adPwd3Edit">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="adminMenu">
    <property name="title">
     <string>管理</string>
    </property>
    <addaction name="adDishesAction"/>
    <addaction name="adPeopleAction"/>
    <addaction name="adOrderAction"/>
   </widget>
   <widget class="QMenu" name="moneyMenu">
    <property name="title">
     <string>营业额</string>
    </property>
    <addaction name="adDayAction"/>
    <addaction name="adMonthAction"/>
    <addaction name="adDateAction"/>
   </widget>
   <widget class="QMenu" name="pwdMenu">
    <property name="title">
     <string>密码管理</string>
    </property>
    <addaction name="adPwdAction"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>菜单</string>
    </property>
    <addaction name="adCancelAction"/>
   </widget>
   <addaction name="adminMenu"/>
   <addaction name="moneyMenu"/>
   <addaction name="pwdMenu"/>
   <addaction name="menu"/>
  </widget>
  <action name="adDishesAction">
   <property name="text">
    <string>菜品管理</string>
   </property>
  </action>
  <action name="adPeopleAction">
   <property name="text">
    <string>人员管理</string>
   </property>
  </action>
  <action name="adOrderAction">
   <property name="text">
    <string>订单管理</string>
   </property>
  </action>
  <action name="adDayAction">
   <property name="text">
    <string>当日营业额</string>
   </property>
  </action>
  <action name="adMonthAction">
   <property name="text">
    <string>当月营业额</string>
   </property>
  </action>
  <action name="adDateAction">
   <property name="text">
    <string>全部营业额</string>
   </property>
   <property name="toolTip">
    <string>按日期统计</string>
   </property>
  </action>
  <action name="adPwdAction">
   <property name="text">
    <string>修改密码</string>
   </property>
  </action>
  <action name="adCancelAction">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
